distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -91,10 +91,10 @@ new_builder (capacity=10) = Builder.new capacity
    A vector allows to store an arbitrary number of elements in linear memory. It
    is the recommended data structure for most applications.
 from_polyglot_array : Any -> Vector Any
-from_polyglot_array arr = Vector (Proxy_Polyglot_Array.Proxy_Polyglot_Array arr)
+from_polyglot_array arr = Vector_Data (Proxy_Polyglot_Array.Proxy_Polyglot_Array_Data arr)
 
 ## The basic, immutable, vector type.
-type Vector
+type Vector a
```

Comment:
:scream: we have that already?

Reply:
yes, it's a no-op, but you should be writing it to future-proof the code